tests/ui/dis/ptr_copy.normal.stderr

```diff
@@ -1,7 +1,7 @@
 error: Cannot memcpy dynamically sized data
-    --> $CORE_SRC/intrinsics.rs:2431:9
+    --> $CORE_SRC/intrinsics.rs:2449:9
      |
-2431 |         copy(src, dst, count)
+2449 |         copy(src, dst, count)
```

Comment:
Maybe we should use the regex "normalization" feature to erase away these line numbers lolsob.